thor/src/protocols/update.rs

```diff
@@ -18,8 +18,8 @@ use serde::{Deserialize, Serialize};
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[derive(Debug)]
 pub struct ShareKeys {
-    R: RevocationPublicKey,
-    Y: PublishingPublicKey,
+    r: RevocationPublicKey,
```

Comment:
>  For `R`, and `Y` the capitalisation implies that it is the 'other' party

We use capital letters to indicate public keys (`Point`s), as opposed to lowercase to indicate secret keys (`Scalar`s).

Reply:
Oh, cool - I misunderstood this. Thanks, will remove this patch.

Reply:
Can you verify this some more please?

We have
```
    y_self: PublishingKeyPair,
    Y_other: PublishingPublicKey,
```

Where
```
pub struct PublishingKeyPair {
    secret_key: Scalar,
    public_key: Point,
}

pub struct PublishingSecretKey(Scalar);
```

Reply:
1. A `PublishingPublicKey` is a newtype around the `Point` type, which is shared between parties when creating and updating a channel. 
2. A `PublishingSecretKey` is a newtype around the `Scalar` type, which is private to each party and is only leaked when one of the parties publishes the corresponding `CommitTransaction`.
3. A `PublishingKeyPair` is a convenience struct to hold both the secret key and the public key. In hindsight I think it would be simpler to remove these `KeyPair` types altogether and just use the `SecretKey` types instead (as you can always derive the public key from the secret key). As an implementation detail I would probably still pre-compute the public keys so that we don't have to compute anything every time we want to get the public key from the secret key.

The paper uses the letters `y` and `Y` to represent publishing secret keys and publishing public keys respectively, so we do the same.

---

I actually don't see any inconsistency in what you're showing me though.